setup.cfg

```diff
@@ -1,2 +1,52 @@
 [bdist_wheel]
-universal=1
+universal = 1
+
+[metadata]
+name = vcrpy
+version = 5.1.0
+author = Kevin McCarthy
+author_email = me@kevinmccarthy.org
+license = MIT
+description = Automatically mock your HTTP interactions to simplify and speed up testing
+url = https://github.com/kevin1024/vcrpy
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Environment :: Console
+    Intended Audience :: Developers
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: Implementation :: CPython
+    Programming Language :: Python :: Implementation :: PyPy
+    Topic :: Software Development :: Testing
+    Topic :: Internet :: WWW/HTTP
+    License :: OSI Approved :: MIT License
+
+[options]
+packages = find:
+install_requires =
+    PyYAML
+    wrapt
+    yarl
+    urllib3 <2; python_version <'3.10'
+python_requires = >=3.8
+tests_require =
+    aiohttp
+    boto3
+    httplib2
+    httpx
+    pytest
+    pytest-aiohttp
+    pytest-httpbin
+    requests>=2.16.2
+    tornado
+    Werkzeug==2.0.3
```

Comment:
Also here: We're losing an important comment here for why we have this arguably unexpected dependency in place. Please bring it back.

Reply:
sorry. Also restored.